script.js

```diff
@@ -290,6 +293,15 @@ function initializeDragAndDrop () {
     beforeStop: function (event, ui) {
     
     },
+    items: ":not(.heading)",
+    start: function(event, ui) {
+      let isChore = $(ui.item[0]).is(".chore");
+      if (isChore) {
+        $(".ready-for-review").toggleClass("warn", 700, "easeOutSine", function() {
+          $(".ready-for-review").removeClass("warn", 700);
```

Comment:
I'd move the `removeClass` to a `stop` event or similar to keep the `warn` class on there the whole time you're dragging.